packages/react-components/react-input/etc/react-input.api.md

```diff
@@ -26,7 +26,7 @@ export type InputOnChangeData = {
 export type InputProps = Omit<ComponentProps<Partial<InputSlots>, 'input'>, 'children' | 'defaultValue' | 'onChange' | 'size' | 'type' | 'value'> & {
     children?: never;
     size?: 'small' | 'medium' | 'large';
-    appearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter';
+    appearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter' | 'filled-darker-shadow' | 'filled-lighter-shadow';
```

Comment:
are these new names standardized across controls? feels like our appearance list is a drop zone for any new visual variant design asks for.

Reply:
okay, i see this is work across multiple controls

https://github.com/microsoft/fluentui/issues/24642

Reply:
I'm in the process of adding it to all input components which are the only components that have filled-lighter/filled-darker. So all of them will have the same names.